drivers/rn2xx3/rn2xx3_getset.c

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_rn2xx3
+ * @{
+ * @file
+ * @brief       Implementation of get and set functions for RN2XX3
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#include <string.h>
+#include <errno.h>
+
+#include "assert.h"
+#include "rn2xx3.h"
+#include "rn2xx3_internal.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+char cmd[32];
```

Comment:
What is this "char cmd[32]", is it left over from debugging/developing?

Reply:
it was a left over, I removed it.